pep-0012.rst

```diff
@@ -438,43 +434,92 @@ Hyperlinks
 ----------
 
 When referencing an external web page in the body of a PEP, you should
-include the title of the page in the text, with either an inline
-hyperlink reference to the URL or a footnote reference (see
-`Footnotes`_ below).  Do not include the URL in the body text of the
-PEP.
+include the title of the page or a suitable description in the text, with
+either an inline hyperlink or a separate explicit target with the URL.
+Do not include bare URLs in the body text of the PEP, and use HTTPS
+links wherever available.
 
 Hyperlink references use backquotes and a trailing underscore to mark
 up the reference text; backquotes are optional if the reference text
-is a single word.  For example::
+is a single word.  For example, to reference a hyperlink target named
+``Python web site``, you would write:
+
+.. code-block:: rst
+
+    In this paragraph, we refer to the `Python web site`_.
+
+which renders as:
 
     In this paragraph, we refer to the `Python web site`_.
 
-An explicit target provides the URL.  Put targets in a References
-section at the end of the PEP, or immediately after the reference.
+.. _Python web site: https://www.python.org/
+
+If you intend to only reference a link once, and want to define it inline
+with the text, insert the link into angle brackets (``<>``) after the text
+you want to link, but before the closing backtick, with a space between the
+text and the opening backtick. You should also use a double-underscore after
+the closing backtick instead of a single one, which makes it an anonymous
+reference to avoid conflicting with other target names. For example:
+
+.. code-block:: rst
+
+    Visit the `website <https://www.python.org/>`__ for more.
+
+which displays as:
+
+    Visit the `website <https://www.python.org/>`__ for more.
+
+.. _Python web site: https://www.python.org/
+
+If you want to use one link multiple places with different linked text,
+or want to ensure you don't have to update your link target names when
+changing the linked text, include the target name within angle brackets
+following the text to link, *with an underscore after the target name
+but before the closing angle bracket* (or the link **will not work**).
+
+For example:
+
+.. code-block:: rst
+
+    For further examples, see the `documentation <pydocs_>`_.
+
+which like the above, renders as:
+
+    For further examples, see the `documentation <pydocs_>`_.
+
+.. _pydocs: https://docs.python.org/
```

Comment:
Same question: should this line also be part of the RST code block so it's visible as an example?

Reply:
Like the other one, it is listed in the RST code block immediately below that discusses link targets.